src/integration-test/java/uk/ac/ebi/tsi/portal/api/team/TeamRestControllerIT.java

```diff
@@ -86,21 +99,112 @@ public void can_get_a_team() throws Exception{
 
 			logger.info("Response "  +response);
 		}
-		
-		
-		public void can_create_a_team() throws Exception{
-			Team team = new Team();
-			team.setName(teamName);
-			String json = mapper.writeValueAsString(team);
+
+	@Test
+	public void team_owner_can_add_team_contact_emails() throws Exception {
+
+		String emails = "contact1@ebi,contact2@ebi";
+		mockMvc.perform(
+				put("/team/" + "test-team1" + "/contactemail/")
+						.headers(createHeaders(getToken(ajayUserName, ajayPassword)))
+						.contentType(MediaType.APPLICATION_JSON)
+						.content(emails)
+						.accept(MediaType.APPLICATION_JSON))
+				.andExpect(status().isOk());
+
+	}
+
+	@Test
+	public void team_owner_can_remove_team_contact_emails() throws Exception {
+
+		String deleteURL = "/team/" + "test-team1" + "/contactemail/" + "contact@ebi";
+		mockMvc.perform(
+				delete(deleteURL)
+						.headers(createHeaders(getToken(ajayUserName, ajayPassword)))
+						.contentType(MediaType.APPLICATION_JSON)
+						.accept(MediaType.APPLICATION_JSON))
+				.andExpect(status().isNoContent());
+	}
+
+	@Test
+	public void non_team_owner_cannot_add_team_contact_emails() throws Exception {
+
+		String emails = "contact1@ebi,contact2@ebi";
+		try {
 			mockMvc.perform(
-					post("/team")
-					.headers(createHeaders(token))
-					.contentType(MediaType.APPLICATION_JSON)
-					.content(json)
-					.accept(MediaType.APPLICATION_JSON))
-			.andExpect(jsonPath("name").value(teamName))
-			.andReturn();
+					put("/team/" + "test-team1" + "/contactemail/")
+							.headers(createHeaders(token))
+							.contentType(MediaType.APPLICATION_JSON)
+							.content(emails)
+							.accept(MediaType.APPLICATION_JSON))
+			;
+		} catch (NestedServletException e) {
+			assertEquals(e.getCause().getClass(), TeamAccessDeniedException.class);
 		}
+	}
+
+	@Test
+	public void non_team_owner_cannot_remove_team_contact_emails() throws Exception {
+
+		String deleteURL = "/team/" + "test-team1" + "/contactemail/" + "contact@ebi";
+		try {
+			mockMvc.perform(
+					post(deleteURL)
+							.headers(createHeaders(token))
+							.contentType(MediaType.APPLICATION_JSON)
+							.accept(MediaType.APPLICATION_JSON))
+			;
+		} catch (NestedServletException e) {
+			assertEquals(e.getCause().getClass(), TeamAccessDeniedException.class);
+		}
```

Comment:
I hope we are getting a proper 403 http error response and not a NestedServletException? What is going on here?